requirements.txt

```diff
@@ -0,0 +1,4 @@
+flask==0.10
```

Comment:
Did you find that ``flask-track-usage`` doesn't work with the latest Flask release (0.11)? If so, that's a bug and I should fix it :laughing:.